aporia.nim

```diff
@@ -1686,23 +1686,58 @@ proc extraBtn_Clicked(button: PButton, user_data: Pgpointer) =
 proc goLine_Changed(ed: PEditable, d: Pgpointer) =
   var line = win.goLineBar.entry.getText()
   var lineNum: BiggestInt = -1
-  if parseBiggestInt($line, lineNum) != 0:
+  var column: BiggestInt = 0
+  var validLine = 0
+  var validColumn = 0
+  var columnEntered = false
```

Comment:
I think we can simplify these variables significantly. I'm not sure supporting negative line numbers is worthwhile, how often is it actually useful? Dropping support for them would allow you to use the ``-1`` value as "Invalid line/column specified".